test/src/RunTrailRendererStress.cpp

```
@@ -0,0 +1,672 @@
+  if (StringIsEqualIgnoreCase(value, "long"))
+    return TrailSettings::Length::LONG;
+  if (StringIsEqualIgnoreCase(value, "full"))
+    return TrailSettings::Length::FULL;
+  return std::nullopt;
+}
+
+static bool
+ParseUnsignedOption(const char *arg, const char *prefix,
+                    unsigned &value) noexcept
+{
+  if (!StringStartsWith(arg, prefix))
```

Cppcheck (2.21.0)

[error] Dereference of an invalid iterator

(derefInvalidIterator)